src/SonarScanner.MSBuild.PreProcessor/SonarScanner.MSBuild.PreProcessor.csproj

```diff
@@ -9,6 +9,9 @@
      -->
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(SignAssembly)'=='true'">
+    <DefineConstants>SignAssembly</DefineConstants>
```

Comment:
This will delete all other preexisting values in `<DefineConstants>`
There should have been
```
<DefineConstants>$(DefineConstants);SignAssembly</DefineConstants>
```
see
https://github.com/SonarSource/sonar-dotnet/blob/20e425fc1b21ecdc5987ea7e9b3fe5daff6782f3/analyzers/Directory.Build.targets#L15-L17

Reply:
Should be fixed in : https://github.com/SonarSource/sonar-scanner-msbuild/pull/1177